dspace-api/src/main/java/org/dspace/access/status/AccessStatusHelper.java

```diff
@@ -39,4 +42,17 @@ public String getAccessStatusFromItem(Context context, Item item, LocalDate thre
      * @throws SQLException An exception that provides information on a database access error or other errors.
      */
     public String getEmbargoFromItem(Context context, Item item, LocalDate threshold) throws SQLException;
+
+    /**
+     * Calculate the access status for the bitstream.
+     *
+     * @param context the DSpace context
+     * @param bitstream the bitstream
+     * @param threshold the embargo threshold date
+     * @param type the type of calculation
+     * @return a pair with an access status value and the availability date
+     * @throws SQLException An exception that provides information on a database access error or other errors.
+     */
+    public Pair<String, LocalDate> getAccessStatusFromBitstream(Context context,
```

Comment:
Typing this as `Pair` feels a little unsustainable, we may need to update all of the interfaces/usages again if things change later.

Could you make an `AccessStatus` object to represent this?